web/src/compose_recipient.ts

```diff
@@ -298,6 +301,49 @@ export function handle_middle_pane_transition(): void {
     }
 }
 
+export function maybe_show_go_to_conversation_button_intro_tooltip(): void {
+    // The tooltip is shown only if:
+    // The tooltip hasn't been shown before.
+    // The user has confirmed a recipient change to a topic/DM that
+    // is not in the current view.
+    // The "Go to conversation" button is active.
+    // There are no compose banners currently visible.
+
+    if (
+        !onboarding_steps.ONE_TIME_NOTICES_TO_DISPLAY.has("intro_go_to_conversation_button_tooltip")
+    ) {
+        return;
+    }
+
+    if (
+        !$(".conversation-arrow").hasClass("narrow_to_compose_recipients") ||
+        $("#compose_banners .main-view-banner").length > 0
+    ) {
+        return;
+    }
+
+    const $go_to_conversation_button = $(".conversation-arrow");
+    const element: tippy.ReferenceElement = the($go_to_conversation_button);
+
+    const tippy_instance =
+        element._tippy ??
+        tippy.default(element, {
+            trigger: "manual",
```

Comment:
Let's add `showoncreate`  and use `content` attribute to set the content. Then you won't need to do `setContent` and `show`.